routes/courseRoutes.js

```diff
@@ -54,6 +55,21 @@ router.post("/courses", async (req, res) => {
   }
 });
 
+router.get("/course/getHome", async (req, res) => {
```

Comment:
This URI doesn't match the standard we are following:
- The request type is spelled out
- It contains capitalization

Proposed change: `/course/home`